spec/tooltip-manager-spec.coffee

```diff
@@ -149,6 +160,29 @@ describe "TooltipManager", ->
       it "hides the tooltips", ->
         manager.add element, title: "Title"
         hover element, ->
-          expect(document.body.querySelector(".tooltip")).toBeDefined()
+          expect(document.body.querySelector(".tooltip")).not.toBeNull()
```

Comment:
I think it's better to write `.not.toBeNull` because `.toBeDefined` is `.not.toBeUndefined` which is also true if it's `null`.